### PR TITLE
Add find-tabular-errors notebook

### PR DESCRIPTION
**Pull request by cmauck10**

### CODE REVIEW

File: find_tabular_errors/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cleanlab==2.2
```
Comment (jwmueller) [inline find_tabular_errors/requirements.txt]:
```suggestion
```

Comment (jwmueller) [reply find_tabular_errors/requirements.txt]: @cmauck10 don't need cleanlab from requirements for all examples, we always assume correct version is already installed

File: README.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ To quickly learn the basics of running cleanlab on your own data, we recommend f
...
 | 13  | [entity_recognition](entity_recognition/entity_recognition_training.ipynb)                                             | Train Transformer model  for Named Entity Recognition and produce out-of-sample `pred_probs` for **cleanlab.token_classification**.      |
 | 14  | [cnn_coteaching_cifar10](cnn_coteaching_cifar10)                                               | Train a [Convolutional Neural Network](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/cifar_cnn.py) on noisily labeled Cifar10 image data using cleanlab with [coteaching](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/coteaching.py).  |
+| 15 | [find_tabular_errors](find_tabular_errors/find_tabular_errors.ipynb) | Find label errors in a tabular [Student Grades Demo](https://github.com/cleanlab/s/blob/master/student-grades-demo.csv) dataset using an XGBoost classifier.
```
Comment (jwmueller) [inline README.md]:
```suggestion
| 15 | [find_tabular_errors](find_tabular_errors/find_tabular_errors.ipynb) | Handle mislabeled [tabular data](https://github.com/cleanlab/s/blob/master/student-grades-demo.csv) to improve a XGBoost classifier.
```

**Review by jwmueller — APPROVED** [review]
Good stuff!

**Comment by jwmueller** [comment]
There is one remaining issue though.
Screen Shot 2023-01-26 at 10 31 32 PM

**Comment by cmauck10** [comment]
What does this mean? How do I go about changing the kernel? @jwmueller

**Comment by huiwengoh** [comment]
> What does this mean? How do I go about changing the kernel? @jwmueller

In jupyter notebook, go to `edit` > `edit notebook metadata` and ensure that the the "display_name" field in the json says "Python 3 (ipykernel)"

Apologies this is unclear, will add more explicit instructions to dev.md for future developers

**Comment by cmauck10** [comment]
When I opened the notebook in Jupyter, the kernel was the correct default. It also had a colab attribute (downloaded from colab originally), so I removed that. Hope it passes now.

**Review by jwmueller — APPROVED** [review]
LGTM, I will add to developer instructions about needing to have colab badge in its own cell

**Pull request merged**